proxy/src/registry.rs

```diff
@@ -10,7 +10,11 @@ use std::convert::TryFrom;
 use std::fmt;
 use std::str::FromStr;
 
-use radicle_registry_client::{self as protocol, ClientT, CryptoPair};
+use radicle_registry_client::{
+    self as protocol, ClientT, CryptoPair, REGISTER_MEMBER_DEPOSIT, REGISTER_ORG_DEPOSIT,
+    REGISTER_PROJECT_DEPOSIT, REGISTER_USER_DEPOSIT,
+};
```

Comment:
```suggestion
use radicle_registry_client::{self as protocol, ClientT, CryptoPair};
```
Usually aim to import the module, and if needed traits. Left suggestions that can be applied together.